lib/plug/router.ex

```diff
@@ -218,11 +218,11 @@ defmodule Plug.Router do
 
   This is particularly useful when used with `Plug.Builder.builder_opts/0`.
   `builder_opts/0` allows us to pass options received when initializing
-  `AppRouter` to a specific plug, such as dispatch itself. So if instead of
+  `AppRouter` to a specific plug, such as dispatch itself. So if instead of:
```

Comment:
Btw, the previous shape was also correct. You can skip the `:` which is useful when showing contiguous examples. That's why the next line doesn't start with uppercase. :)

Reply:
Yeah. It’s fine for me too :)